tools/pipelines/test-perf-benchmarks-using-built-artifacts.yml

```diff
@@ -0,0 +1,376 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
```

Comment:
I assume this pipeline replace an existing pipline?

Reply:
Yes it will replace this one: https://github.com/microsoft/FluidFramework/blob/main/tools/pipelines/test-perf-benchmarks.yml (https://dev.azure.com/fluidframework/internal/_build?definitionId=96&_a=summary)

Reply:
@curtisman does that mean I should actually replace the current pipeline with this PR? 😄

Reply:
Is there a reason not to?

Reply:
I think it's fine to replace the existing one. And for that, just make the changes directly in the existing pipeline's yml definition.

Reply:
Done!